### PR TITLE
Extend StatsBase.describe to handle Nulls

### PR DESCRIPTION
**Pull request by cjprybol**
For merging into #66

### CODE REVIEW

File: test/utils.jl [hunk 1]
```diff
@@ -110,5 +110,46 @@ module TestUtils
...
+        describe(io, nulls(Char, 3))
+        @test String(take!(io)) == """
+                                   Summary Stats:
+                                   Length:         3
+                                   Type:           Char
+                                   Number Unique:  1
+                                   Number Missing: 3
+                                   % Missing:      100.000000
+                                   """
+        describe(io, Vector{?Int}(1:3))
```
Comment (ararslan) [inline test/utils.jl]: This last one shouldn't be necessary, right?

File: src/abstractdatatable/abstractdatatable.jl [hunk 2]
```diff
@@ -378,6 +378,32 @@ function StatsBase.describe(io, dt::AbstractDataTable)
         println(io, )
     end
 end
+function StatsBase.describe{T <: Real}(io::IO, X::Vector{Union{T, Null}})
+    nullcount = sum(isnull, X)
```
Comment (ararslan) [inline src/abstractdatatable/abstractdatatable.jl]: I usually prefer `count` for this sort of thing, but `sum` is equivalent so it doesn't really matter

File: test/utils.jl (hunk 1 above)
Comment (cjprybol) [reply test/utils.jl]: Thanks!

File: test/utils.jl [hunk 3]
```diff
@@ -110,5 +110,45 @@ module TestUtils
...
+        @test String(take!(io)) == """
+                                   Summary Stats:
+                                   Mean:           2.000000
+                                   Minimum:        1.000000
+                                   1st Quartile:   1.500000
+                                   Median:         2.000000
+                                   3rd Quartile:   2.500000
+                                   Maximum:        3.000000
+                                   Length:         3
+                                   Type:           Int64
```
Comment (nalimilan) [inline test/utils.jl]: Should be `$Int`.

**Comment by nalimilan** [comment]
Thanks. I guess @quinnj should just pick the commit into #66, since this PR cannot really be merged on its own.

**Comment by oxinabox** [comment]
This will fix: https://github.com/JuliaData/DataTables.jl/issues/60
right?

**Comment by nalimilan** [comment]
#60 is already fixed on DataTables master.

**Comment by cjprybol** [comment]
superseded by #83

**Pull request closed without merging**